development/releasing.md

```diff
@@ -2,29 +2,42 @@
 
 ## Webapp
 
-When all the required issues are fixed in master it's time to put together a release.
+### Planning
 
-1. Pick a version number. We use [semver](http://semver.org) so if there are breaking changes increment the major, otherwise if there are new features increment the minor, otherwise increment the service pack. Breaking changes in our case relate to updated software requirements (egs: CouchDB, node, minimum browser versions) or broken backwards compatibility in an api.
-2. If releasing a new major or minor...
-  - Update [google-libphonenumber](https://www.npmjs.com/package/google-libphonenumber) to the latest version in api, sentinel, and webapp and commit to `master` and push. This ensures we're up to date with the latest in phone number validation.
-```bash
-npm install --save google-libphonenumber@latest
+1. Create an [organisation wide project](https://github.com/orgs/medic/projects?query=is%3Aopen+sort%3Aname-asc). We use [semver](http://semver.org) so if there are breaking changes increment the major, otherwise if there are new features increment the minor, otherwise increment the service pack. Breaking changes in our case relate to updated software requirements (egs: CouchDB, node, minimum browser versions) or broken backwards compatibility in an api.
+2. Add all the issues to be worked on to the project. Ideally each minor release will have one or two features, a handful of improvements, and plenty of bugs.
+
+### Development
+
+1. When development is ready to begin one of the developers should be nominated as a Release Manager, who will be resonsible for making sure the following tasks are completed though not necessarily completing them.
+2. Set the version number in package.json and package-lock.json.
+3. For new major and minor releases update all dependencies that don't have any breaking changes. If there are dependencies with easy to fix breaking changes feel free to fix those too. Raise issues to fix the hard breaking changes.
+4. Write an update in the weekly Product Team call agenda summarising development and acceptance testing progress and identifying any blockers.
+
+### Releasing
+
+Once all issues have passed acceptance testing and have been merged into `master` release testing can begin.
+
+1. Create a new release branch from `master` named `<major>.<minor>.x` in medic. Notify all developers that any further commits for this release will need to be backported to the branch.
```

Comment:
A sample message to use as a template (like done for the announcement) would be useful.